File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        maxi = 0
        for i in range(0,len(nums)):
            if(nums[i]==1) :
                count+=1
                maxi = max(count,maxi)
            else : count = 0
        return maxi
        

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int max(int a, int b) {
        if (a>=b) {
            return a;
        } 
        else {
            return b;
        }
    }

    public int findMaxConsecutiveOnes(int[] nums) {
        int maxi = 0;
        int count = 0;
        for(int i=0; i<nums.length ; i++) {
            if (nums[i]==1) {
                count++ ;
                maxi = max(count,maxi);
            }
            else {
                count = 0;
            }
        }
        return maxi;
    }
}